acsylla/_cython/statement/bind.pyx

```diff
@@ -111,7 +116,7 @@ cdef inline bind_int64_by_name(CassStatement* statement, bytes name, object valu
 
 # CASS_VALUE_TYPE_FLOAT
 cdef get_float(object value):
-    if isinstance(value, (int, Decimal)) or (isinstance(value, str) and value.isdigit()):
+    if isinstance(value, (int, str, Decimal)):
```

Comment:
I did not see this in my previous reviews, whats the reason on providing support for this implicit casting? Im concerned of asking to the library to make this abstraction.

Reply:
@pfreixes why not? this is pythonic way I don't want think about data type. If my data string  like a duck maybe this is duck? ;)
if it is important to me what I write to the database, I will additionally check the data before inserting, but I don't want `ValueError` when I insert `"3.14"` in  float type row